reference-implementation/lib/writable-stream.js

```diff
@@ -551,11 +535,8 @@ function WritableStreamDefaultWriterClose(writer) {
     return Promise.reject(new TypeError(
       `The stream (in ${state} state) is not in the writable state and cannot be closed`));
   }
-  if (stream._pendingAbortRequest !== undefined) {
-    return Promise.reject(new TypeError('Requested to abort'));
-  }
 
-  assert(state === 'writable');
+  assert(state === 'writable' || state === 'erroring');
```

Comment:
Hmm, what was the rationale to have close() succeed while write() fails when the state is erroring?